submissions/ORybka/index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Patrick+Hand&display=swap" rel="stylesheet">
+  <link rel="shortcut icon" href="./assets/kitty.svg" type="image/svg">
+  <link rel="stylesheet" href="style.css">
+  <title>Hooli-popup</title>
+</head>
+<body>
+  <div class="container">
+    <header>
+      <nav class="navbar">
+        <ul class="menu">
+          <li class="menu-item">
+            <a href="#" >
+              Mail
+            </a>
+          </li>
+          <li class="menu-item">
+            <a href="#" >
+              Images
+            </a>
+          </li>
+          <li class="menu-item">
+            <input class="menu-label btn-open" type="checkbox" id="menu-btn">
+            <a href="#" class="menu-item-link">
```

Comment:
Maybe you don't need to use `a` tag here.

Reply:
@MaxKalashnyk is it possible to keep a tag? I can't solve the problem with this element to be focused without a (or I ask for help, please :cry:

Reply:
I wouldn't recommend using `a` tag here. I can help you to fix your issue.
It seems it happened because you hide the input using `display: none;`. I think it's not an optimal solution here.
I recommend you watching this [video](https://www.youtube.com/watch?v=E6kLaaQFctU&ab_channel=VadimMakeev). It can help to solve your issue with the focus on  applying proper styles for your `input`.

Reply:
Thank you very much for the video! This is perfect!